activerecord/lib/active_record/associations/preloader/association.rb

```diff
@@ -52,20 +52,14 @@ def associated_records_by_owner(preloader)
           end
 
           def owner_keys
-            unless defined?(@owner_keys)
-              @owner_keys = owners.map do |owner|
-                owner[owner_key_name]
-              end
-              @owner_keys.uniq!
-              @owner_keys.compact!
-            end
-            @owner_keys
+            @owner_keys ||= owners_by_key.keys
```

Comment:
This is not a same with before.
Previously `@owner_keys` is an array of raw `owner[owner_key_name]`. But `owners_by_key.keys` is converted owner keys.

Reply:
Yes but converted keys are just converted to a string which doesn't make any difference from what I can tell.